app/scripts/directives/mobileServiceInstanceRow.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+(function() {
+  angular.module('openshiftConsole').component('mobileServiceInstanceRow', {
+    controller: [
+      '$filter',
+      '$routeParams',
+      '$scope',
+      'APIService',
+      'AuthorizationService',
+      'BindingService',
+      'Constants',
+      'DataService',
+      'ListRowUtils',
+      'ServiceInstancesService',
+      MobileServiceInstanceRow
+    ],
+    controllerAs: 'row',
+    bindings: {
+        apiObject: '<',
+        serviceClass: '<',
+        binding: '<',
+        state: '<'
+    },
+    templateUrl: 'views/_mobile-service-instance-row.html'
+  });
+
+  function MobileServiceInstanceRow($filter,
+                              $routeParams,
+                              $scope,
+                              APIService,
+                              AuthorizationService,
+                              BindingService,
+                              Constants,
+                              DataService,
+                              ListRowUtils,
+                              ServiceInstancesService) {
+    var row = this;
+    var serviceInstanceDisplayName = $filter('serviceInstanceDisplayName');
+    row.serviceBindingsVersion = APIService.getPreferredVersion('servicebindings');
+    row.state = row.state || {};
+    var annotationSpace = "org.aerogear.binding." + this.apiObject.metadata.name;
+    var watches = [];
+    _.extend(row, ListRowUtils.ui);
```

Comment:
What does this do?

Reply:
this is copied from serviceInstanceRow, removing it seems to stop the expanding and collapsing from working, and the kebab menu.

Reply:
:+1: